packages/stryker/src/MutatorOrchestrator.ts

```diff
@@ -26,7 +26,7 @@ export default class MutatorOrchestrator {
   /**
    * @param reporter - The reporter to report read input files to
    */
-  public constructor(private reporter: StrictReporter) {
+  public constructor(private reporter: StrictReporter, private options: {}) {
```

Comment:
I would make options be of type `StrykerOptions` here and add the option you need to `StrykerOptions`. `{}` means of type `empty object` and that is normally not what you want.